docs/ui/components/Collapsible/index.tsx

```diff
@@ -0,0 +1,118 @@
+import { css } from '@emotion/react';
+import { borderRadius, iconSize, shadows, spacing, theme } from '@expo/styleguide';
+import React, { PropsWithChildren, ReactNode } from 'react';
+
+import { HEADLINE } from '~/ui/components/Text';
+import { durations } from '~/ui/foundations/durations';
+import { ChevronDownIcon } from '~/ui/foundations/icons';
+
+type CollapsibleProps = PropsWithChildren<{
+  /** The content of the collapsible summary */
+  summary: ReactNode;
+  /** If the collapsible should be rendered "open" by default */
+  open?: boolean;
+  testID?: string;
+}>;
+
+export function Collapsible({ summary, open, testID, children }: CollapsibleProps) {
+  return (
+    <details css={detailsStyle} open={open} data-testid={testID}>
+      <summary css={summaryStyle}>
+        <ChevronDownIcon css={markerStyle} size={iconSize.small} />
+        <HEADLINE tag="span" css={headlineStyle}>
+          {summary}
+        </HEADLINE>
+      </summary>
+      <div css={contentStyle}>{children}</div>
+    </details>
+  );
+}
+
+const detailsStyle = css({
+  overflow: 'hidden',
+  background: theme.background.default,
+  border: `1px solid ${theme.border.default}`,
+  borderRadius: borderRadius.medium,
+  padding: 0,
+
+  '&[open]': {
+    boxShadow: shadows.tiny,
```

Comment:
```suggestion
    boxShadow: shadows.micro,
```

After seeing this, I think i prefer a more subtle shadow.

Reply:
Me too, especially on dark mode